R/utils.R

```diff
@@ -10,7 +10,7 @@ is_windows <- function() .Platform$OS.type == "windows"
 
 is_mac     <- function() Sys.info()[['sysname']] == 'Darwin'
 
-is_linux   <- function() Sys.info()[['sysname']] == 'Linux'
+is_linux   <- function() Sys.info()[['sysname']] %in% c('Linux', 'OpenBSD')
```

Comment:
I don't like the idea of changing this function so that it returns true for OpenBSD because it makes the function name misleading. For example, one of the places where `is_linux()` is used is `is_inside_docker()`, and a Docker container must be running Linux.

I'd prefer adding `is_openbsd()` instead.

Reply:
I agree this is a bit clumsy, and I tend to agree with you about the naming. However, I am not sure if a separate function for each OS would be making things unnecessarily verbose, since at least for the BSDs you have OpenBSD, FreeBSD, NetBSD, DragonflyBSD, although they are all pretty similar. In this case `is_bsd()` might be an option, but would require testing on other OSs.

Reply:
Another issue with `is_unix()` is that Mac OS is also a Unix, but we generally want to treat it differently from other Unixes. I think Mac OS also sort of counts as a BSD, so `is_bsd()` would face the same issue.

It would be nice to have a function like `is_non_mac_unix()`, but with a less-awkward name. But for the practical purposes that you're trying to solve, I think adding `is_openbsd()` is fine.